src/core_plugins/elasticsearch/lib/__tests__/routes.js

```diff
@@ -30,12 +30,11 @@ describe('plugins/elasticsearch', function () {
       return kbnServer.close();
     });
 
-    function testRoute(options) {
+    function testRoute(options, statusCode = 200) {
       if (typeof options.payload === 'object') {
         options.payload = JSON.stringify(options.payload);
       }
 
-      const statusCode = options.statusCode || 200;
       describe(format('%s %s', options.method, options.url), function () {
         it('should should return ' + statusCode, function (done) {
           kbnTestServer.makeRequest(kbnServer, options, function (res) {
```

Comment:
We can not pass `option.statusCode` in the request, as Joi was throwing an error.